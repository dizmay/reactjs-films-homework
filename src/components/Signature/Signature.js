import React from 'react';
import PropTypes from 'prop-types';
import styles from './Signature.scss';

const Signature = ({ name }) => (
  <div className={styles.sign}>
    <h1>{name}</h1>
  </div>
);

Signature.defaultProps = {
  name: null,
};

Signature.propTypes = {
  name: PropTypes.string,
};

export default Signature;
