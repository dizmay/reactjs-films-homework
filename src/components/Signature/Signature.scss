.sign {
  h1 {
    color: lime;
  }
}
