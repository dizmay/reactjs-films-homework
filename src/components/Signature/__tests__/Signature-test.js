import React from 'react';
import renderer from 'react-test-renderer';
import Signature from '../Signature';

it('renders correctly', () => {
  const tree = renderer.create(<Signature />).toJSON();
  expect(tree).toMatchSnapshot();
});
