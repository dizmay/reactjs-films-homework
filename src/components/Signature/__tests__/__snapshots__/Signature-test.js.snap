// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sign"
>
  <h1 />
</div>
`;
